curator-framework/src/main/java/org/apache/curator/framework/api/RemoveWatchesBuilder.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.curator.framework.api;

```

Comment:
Looking at the local case again, I think it's actually just client.getZookeeper() that needs to be used. I notice that this is how it's used in all of the performBackgroundOperation calls, but there are no checks for null pointers in this case. Is there a reason for this?